ironfish-cli/src/commands/chain/genesisblock.ts

```diff
@@ -79,7 +78,7 @@ export default class GenesisBlockCommand extends IronfishCommand {
       allocations: [
         {
           publicAddress: account.publicAddress,
-          amount: CurrencyUtils.decodeIron(GENESIS_SUPPLY_IN_IRON),
+          amount: CurrencyUtils.decodeIron(node.chain.consensus.parameters.genesisSupplyInIron),
```

Comment:
I think what this should probably do eventually is take `genesisSupplyInIron` as a flag to the command, then output a network definition file. Right now, it looks like there's sort of a chicken-egg problem where you need a network definition file (including a genesis block) to generate a genesis block.

I don't think this has to change in this PR though, it seems like it'll still work to assemble a new genesis block.

Reply:
Ok I added a task for fixing this command

Reply:
Added a `genesisSupplyInIron` flag to the command. However, as we discussed I'm going to keep the output as a genesis block and not a network definition file. Let me know if you'd still like me to change this to output a network definition file.